tower-grpc/src/generic/server/streaming.rs

```diff
@@ -44,12 +44,14 @@ where
             Ok(Async::NotReady) => return Ok(Async::NotReady),
             Err(status) => {
                 // Construct http response
-                let mut response = Response::new(Encode::error(status)).into_http();
-                // Set the content type
+                let mut response = Response::new(Encode::empty()).into_http();
+
+                // Set the content type and add the gRPC error codes
                 response.headers_mut().insert(
```

Comment:
Excuse the late question but I wonder if we should explicitly encode `Content-length: 0`?  Or can we trust something else to do that?

Reply:
@seanmonstar ^ thoughts?

Reply:
I don't see anything explicitly within the gRPC spec for this so not sure if this is right or what hyper might add.